lib/packages/docutils/rst.nim

```diff
@@ -1021,6 +1021,16 @@ proc fixupEmbeddedRef(n, a, b: PRstNode) =
   for i in countup(0, sep - incr): a.add(n.sons[i])
   for i in countup(sep + 1, n.len - 2): b.add(n.sons[i])
 
+proc whichRole(sym: string): RstNodeKind =
```

Comment:
just a question: is there a difference between prefix and postfix role, and if not, is there a benefit of allowing both (besides the benefit of conforming to the spec)? (the flip argument being to avoid 2 ways to do the same thing)

Reply:
no, there is no slightest difference, they are identical.

I think that using postfix role should remain preferred. This can be included to contributing.rst. 

The only benefit is being able to parse `*.rst` from other implementations.

Reply:
and IMHO it's not confusing that a role marker can appear from either side. So it seems harmless to have.

Reply:
> The only benefit is being able to parse *.rst from other implementations.

ok convincing; 

> This can be included to contributing.rst.

please yes; in this or followup PR

Reply:
done.